index.js

```diff
@@ -1,111 +1,111 @@
-var path = require('path'),
-  fs = require('fs'),
-  jade = require('jade'),
-  url = require('url'),
-  assert = require('assert');
-
-var defaultOptions = {
-  // Jade options
-  jade: {},
-  // Serve index.jade if http://someurl/example/ is requested
+const _ = require('lodash');
```

Comment:
Can we do away with this and:

assigns can be `{...oldProps, ...newProps}`
includes can be `[].indexOf(val) != -1`
startsWith can be `string[0] == something`

Reply:
okay I can do that